<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_user</class>
 <widget class="QDialog" name="Dialog_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>448</y>
     <width>264</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: The data length is less than or equal to 16 characters</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>148</y>
     <width>264</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>User factor2：</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>f0：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_f2_0"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_w2">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>f1：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_f2_1"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_r2">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>f2：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_f2_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>f3：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_f2_3"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_c2">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>288</y>
     <width>264</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>User factor3：</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>f0：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_f3_0"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_w3">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>f1：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_f3_1"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_r3">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>f2：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_f3_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>f3：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_f3_3"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_c3">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>264</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>User factor1：</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>f0：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_f1_0"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_w1">
      <property name="text">
       <string>Write</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>f1：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_f1_1"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_r1">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>f2：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_f1_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>f3：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_f1_3"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_c1">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
